docs/user_stories/topology.md

```diff
@@ -0,0 +1,100 @@
+The second voltage level VL2 has a single busbar section BBS3, a line LN and is connected to voltage level VL1 through transformer TR. 
+Below is a diagram of the substation:
+
+![Node breaker topology](./images/nodeBreakerTopology.svg){: width="50%" .center-image}
+
+The node/breaker topology model is stored inside the voltage level as a graph, in which the vertices correspond to connection nodes and the edges correspond to switches. 
+The next diagram shows how to map the substation topology to a graph.
+
+![Node breaker graph](./images/nodeBreakerTopologyGraph.svg){: width="50%" .center-image}
+
+- Each voltage level has its own topology graph. Voltage level VL1 has 8 connection nodes. Generator GN is connected to node 1, load LD to node 5, busbar sections BBS1 and BBS2 to node 3 and 4. 2, 6 and 7 are internal connection nodes.
+- Voltage level VL2 has 3 nodes, line LN is connected to node 1, busbar section BBS3 to node 2.
+- Transformer TR is connected to node 8 of voltage level 400 KV and node 3 of voltage level 225 KV.
+- Plain edges represent closed switches.
+- Dashed edges represent open switches.
+- Green edges will disappear during the bus/breaker topology computation whereas pink edges (like 3<->4 in this case) will be retained, whatever their state (see [switches](switch.md)).
+
+## Bus/breaker view
+
+The following diagram shows the computed bus/breaker topology. 
+Compared to the node/breaker topology, only the equipements (GN, LD, TR, LN) and the switches flagged as retained (BR3) remain in the network description.
+The equipments are now connected through buses (B1, B2 and B3).
+
+![Bus breaker graph](./images/busBreakerTopology.svg){: width="50%" .center-image}
+
+We can switch to a bus/breaker view on the substation voltage level VL1. 
+VL1 contains 2 buses in the bus/breaker view. 
+The first bus connects the nodes 1, 2, 3, 5 and 6, and consequently the generator GN and the load LD. 
+Note that the load LD does not belong to the connected component. 
+The second bus connects nodes 4, 7 and 8. Note that VL1 contains only one switch in the bus/breaker view (BR3). 
+Here are the corresponding prints in the tutorial:
+
+````
+Bus: VL1_1
+	 Generators: GN
+Bus: VL1_4
+	 T2W: TR
+Bus: VL1_5
+	 Loads: LD, connected component: null
+Switch: BR3
+Bus: VL2_1
+	 T2W: TR
+````
+
+## Bus view
+
+The following diagram shows the computed bus topology. Compared to bus/breaker topology, there is no switches anymore. Only remains equipements (GN, TR, LN) connected through buses. LD is not connected.
+
+![Bus graph](./images/busTopology.svg){: width="50%" .center-image}
+
+We can switch to a bus view on substation voltage level VL1. VL1 contains 1 buses in the bus view. This bus connects all the equipements of voltage level VL1. Here are the corresponding prints in the tutorial:
+
+````
+Bus:VL1_1
+	 Generators: GN
+	 T2Ws: TR
+Bus:VL2_1
+	 T2Ws: TR
```

Comment:
Same remark here about the separation of VL1 and VL2

Reply:
Same answer as before.

Reply:
Same remark then 😉